alws/build_planner.py

```diff
@@ -134,6 +134,8 @@ async def create_build_repo(
             debug=is_debug,
             platform=platform,
         )
+        if repo_type == 'rpm':
+            repo.platform_id = platform.id
```

Comment:
What is the point of that if we already passed a platform object ?

Reply:
Because the filter doesn't work properly. Looks like a bug. May be it's already fixed in SQLALchemy 2.0

Reply:
I think that @maccelf meant that you are changing `platform_id` to the same id because the platform object is still the same

Reply:
as I remember it can be empty

Reply:
`repo.platform_id` would be empty until you make a flush or commit, if I'm not mistaken
if you are using `platform_id` in your changes, then I guess it would be better to pass `platform_id` instead of `platform` object during repo creation